src/site/stages/build/process-cms-exports/schema-validation.js

```diff
@@ -36,7 +36,7 @@ const validateEntityFactory = schemasDir => {
       if (!missingSchemas.has(contentModelType)) {
         missingSchemas.add(contentModelType);
         // eslint-disable-next-line no-console
-        console.warn(`Missing schema for ${contentModelType}`);
+        console.warn(`Missing ${schemaType} schema for ${contentModelType}`);
```

Comment:
I was getting some `Missing schema for...` and thought I'd be more specific about _which_ schema was missing.